open-machine-learning-jupyter-book/ml-fundamentals/regression/Univariate-linear-regression.ipynb

```diff
@@ -0,0 +1,338 @@
+{
```

Comment:
<p>As a teaching course, we might need a slightly more theoretical introduction and pedagogical content, and it's a bit abrupt to go straight into the code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/747/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>